Source/MLXNN/Cache.swift

```diff
@@ -5,7 +5,7 @@ import Foundation
 /// Simple cache for holding prepared MLXArrays, etc.
```

Comment:
Is this still used anywhere?

Reply:
It was used in RoPE, but that is replaced with MLXFast.  It looks like it is still in ALiBi

Reply:
Ah I forgot about ALiBi.. makes sense.